Snowflake.Data.Tests/UnitTests/SFEnvironmentTest.cs

```diff
@@ -0,0 +1,35 @@
+using NUnit.Framework;
+using Snowflake.Data.Core;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Versioning;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    [TestFixture]
+    class SFEnvironmentTest
+    {
+        [Test]
+        public void TestRuntimeExtraction()
+        {
+            // Arrange
+            var targetFrameworkAttribute = Assembly.GetExecutingAssembly().GetCustomAttributes(true).ToList().OfType<TargetFrameworkAttribute>().First();
```

Comment:
the previous version:
```
...
#elif NET6_0
            expectedVersion = "6.0";
#elif NET7_0
            expectedVersion = "7.0";
#elif NET8_0
            expectedVersion = "8.0";
...
```

was very readable. Current version of tests does not give us so much readability.
Looking at this I'm not sure what the result will be under framework471 or net6.0.
Maybe you could restore the previous way of checking it in tests?
I'm not sure what is the problem here - if it is about that the version name has too much details e.g. 6.0.1 instead of 6.0 we could compare in tests only.
The problem is that if all these complicated methods like `Assembly.GetExecutingAssembly()` or `Runtime...` would return something which does not make sense (for example an empty string) then the test would not discover the problem since similar code is in 2 places: the production code and the test code. But if in test code we would compare the result with simple values like (".net", "6.0") and the method of getting the .net details would change, then the test would fail.

Reply:
I agree, it was much easier to understand before

Reverted the change to the previous version